src/parameters/analytic_options.py

```diff
@@ -7,21 +7,16 @@
     "Large Vessel Detection": "LargeVesselDetection",
     "Aircraft Detection": "AircraftDetection",
     "Ground Vehicle Detection": "GroundVehicleDetection",
-    "Car Counting": "CarCounting",
     "Change Detection": "ChangeDetection",
-    "Subsidence Monitoring": "SubsidenceMonitoring",
+    "Ground Displacement": "GroundDisplacement",
     "Object Detection (SPEX)": "SPEX",
 }
 
 # frontend label to backend enum mapping
 cd_map: dict[str, str] = {
-    "Basic Change Detection": "CHANGE_DETECTION",
-    "Object Level Change Detection": "OBJECT_LEVEL",
-    "Port Traffic": "PORT_TRAFFIC",
-    "Parking Lot": "PARKING_LOT",
-    "Construction": "CONSTRUCTION",
-    "Hurricane Damage": "HURRICANE_DAMAGE",
-    "Large Earth": "LARGE_EARTH",
+    "Small Objects": "SMALL_OBJECTS",
+    "Medium Structures": "Medium Structures",
```

Comment:
for consistency, should be change this to MEDIUM_STRUCTURES ? or is there a functionality reason why it should be left like this?

Reply:
great catch, this was a typo!